logicalplan/plan.go

```diff
@@ -20,34 +20,43 @@ var DefaultOptimizers = []Optimizer{
 	MergeSelectsOptimizer{},
 }
 
+type Opts struct {
```

Comment:
Can we include `expr` here too?

Reply:
Since each optimizer takes the `expr` as the first argument, do we need to add it to the query options as well?